src/util/k5test.py

```diff
@@ -621,6 +623,7 @@ def _parse_args():
     _stop_after = _parse_cmdnum('--stop-after', options.stopa)
     _shell_before = _parse_cmdnum('--shell-before', options.shellb)
     _shell_after = _parse_cmdnum('--shell-after', options.shella)
+    _trace = _parse_cmdnum('--shell-after', options.trace)
```

Comment:
`_parse_cmdnum('--trace', ..)` instead of `--shell-after`

Reply:
Fixed, thanks.